pkcs1/src/document/private_key.rs

```diff
@@ -86,8 +89,8 @@ impl ToRsaPrivateKey for RsaPrivateKeyDocument {
 
     #[cfg(feature = "pem")]
     #[cfg_attr(docsrs, doc(cfg(feature = "pem")))]
-    fn to_pkcs1_pem(&self) -> Result<Zeroizing<String>> {
-        let pem_doc = pem::encode_string(PEM_TYPE_LABEL, Default::default(), self.as_der())?;
+    fn to_pkcs1_pem_with_le(&self, line_ending: LineEnding) -> Result<Zeroizing<String>> {
```

Comment:
I know it is too late, but this method name really makes me think of little endian, not line ending unfortuantely

Reply:
Can file an issue about better naming. Here you go:

https://github.com/RustCrypto/utils/issues/558